src/yuv_rgb/transfer.rs

```diff
@@ -149,7 +147,7 @@ fn log100_inverse_oetf(x: f32) -> f32 {
     if x <= 0.0 {
         0.01
     } else {
-        powf(10.0, 2.0 * (x - 1.0))
+        fastapprox::fast::pow(10.0, 2.0 * (x - 1.0))
```

Comment:
Maybe `use fastapprox::fast::pow as powf`? `pow` should (by normal naming conventions) be for double-precision floats (f64) so I think `powf` still makes more sense

Reply:
Makes sense to me. Made the change.